CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## vNext (TBD)
+
+### Enhancements
+* None
+
+### Fixed
+* None
+
+### Internals
+* Foo bar baz
```

Comment:
Was this ment to be committed?

Reply:
No, leftover from testing if it correctly picks up changes